owncloudApp/src/main/java/com/owncloud/android/workers/UploadFileFromContentUriWorker.kt

```diff
@@ -157,14 +199,80 @@ class UploadFileFromContentUriWorker(
 
     private fun uploadDocument() {
         val client = getClientForThisUpload()
-        val requestBody = ContentUriRequestBody(appContext.contentResolver, contentUri).also {
-            it.addDatatransferProgressListener(this)
+
+        val cacheFile = File(cachePath)
+        mimeType = cacheFile.extension
+        fileSize = cacheFile.length()
+
+        val getStoredCapabilitiesUseCase: GetStoredCapabilitiesUseCase by inject()
+        val capabilitiesForAccount = getStoredCapabilitiesUseCase.execute(
+            GetStoredCapabilitiesUseCase.Params(
+                accountName = account.name
+            )
+        )
+        val isChunkingAllowed = capabilitiesForAccount != null && capabilitiesForAccount.isChunkingAllowed()
+        Timber.d("Chunking is allowed: %s", isChunkingAllowed)
```

Comment:
This log could handle more information regarding not only if chunking is allowed but also if the file is big enough to be chunked